resources/views/social-link.blade.php

```diff
@@ -1,8 +1,14 @@
+@props([
+    'iconSize' => '4',
+    'url',
+    'icon',
+])
+
 <a
     href="{{ $url }}"
     target="_blank"
     rel="noopener noreferrer"
     class="transition-default hover:text-theme-secondary-500"
 >
-    @svg($icon, 'w-4 h-4')
+    @svg($icon, sprintf('w-%s h-%s', $iconSize, $iconSize))
```

Comment:
using this will mess with purgecss as it can't figure out the actual classes anymore

Reply:
> using this will mess with purgecss as it can't figure out the actual classes anymore

@ItsANameToo it was used on marketsquare but just changed it for a simpler solution so I removed it